### PR TITLE
moved all sitesusa records to their own file

### PR DESCRIPTION
**Pull request by kyle-jennings**
Note: PRs affecting cloud.gov or Federalist must receive approval from a member of the relevant team.

### CODE REVIEW

**Comment by kyle-jennings** [comment]
@rogeruiz  I renamed the file.  QQ tho, Just triple checking that I don't need to include any of the zone information in this file like the cloud.gv.tf file do it?

https://github.com/18F/dns/blob/deploy/terraform/cloud.gov.tf#L1-L21
and 
https://github.com/18F/dns/blob/deploy/terraform/cloud.gov.tf#L838-L840

**Comment by rogeruiz** [comment]
🤔good points. I'm not sure if the hosted zone is expected to be different that cloud.gov for these entries. If it isn't you should be okay without needing the outputs or to set that variable. I do believe though that each file can reference variables from other files so as long as the outputs are there for `cloud.gov.tf` then it should be good. You can add the output block to your file and name it something relevant to sitesusa so that the output is easier to read when this deploys. The first like shouldn't be necessary. @kyle-jennings

**Comment by rogeruiz** [comment]
@kyle-jennings with a closer look I think it makes more sense to make this it's own hosted zone. So you'll need to make a new resource and a new output. Ping me here for a review.

**Comment by kyle-jennings** [comment]
@rogeruiz Added a zone and output, not sure if this is correct tho.

File: terraform/sitesusa.app.cloud.gov.tf [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+resource "aws_route53_record" "nidanrb_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
+  name = "nidanrb.sitesusa.app.cloud.gov."
+  type = "CNAME"
+  ttl = 60
+  records = ["d3chp99iuj1c5n.cloudfront.net."]
+}
+
+
+output "cloud_gov_ns" {
```
Comment (rogeruiz) [inline terraform/sitesusa.app.cloud.gov.tf]: This named variable should be renamed to `sitesusa_app_cloud_gov_ns` instead of `cloud_gov_ns`.

**Comment by kyle-jennings** [comment]
That was some sloppy copy pasta,  I have updated that variable.  Before we can merge however, it looks like we need you to approve https://github.com/18F/dns/pull/64

**Comment by rogeruiz** [comment]
We need the tests to pass here as well. But thanks for pointing out #64. I don't have to time to merge and monitor it right this second, but I'll try to get to by EOD today.

**Comment by rogeruiz** [comment]
sorry for the delay on this @kyle-jennings. I was originally waiting for #64 to get merged in and then this to get updated with those changes. The placeholder variable needs to be updated. I'll update with line comments.

File: terraform/sitesusa.app.cloud.gov.tf [hunk 2]
```diff
@@ -0,0 +1,384 @@
+/*
+    To keep this repo clean and organized, we are keeping all of the sitesusa
+    dns records here.
+ */
+ resource "aws_route53_zone" "sitesusa_app_cloud_gov_zone" {
```
Comment (rogeruiz) [inline terraform/sitesusa.app.cloud.gov.tf]: This variable name `sitesusa_app_cloud_gov_zone` needs to be referenced in the `zone_id` sections below.

File: terraform/sitesusa.app.cloud.gov.tf [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+   name = "sitesusa.app.cloud.gov."
+   tags {
+     Project = "dns"
+   }
+ }
+
+
+resource "aws_route53_record" "fedramp_sitesusa_app_cloud_gov_cname" {
+  zone_id = "${aws_route53_zone.cloud_gov_zone.zone_id}"
```
Comment (rogeruiz) [inline terraform/sitesusa.app.cloud.gov.tf]:
```
s/${aws_route53_zone.cloud_gov_zone.zone_id}/${aws_route53_zone.sitesusa_app_cloud_gov_zone.zone_id}
```

**Comment by sharms** [comment]
Hi folks,

Cloud.gov does not host CNAME records for tenants under our DNS configurations.  That is managed under the tenants domain name, not under Cloud.gov (or subdomains of Cloud.gov)

**Pull request merged**